pkg/common/configs/config.go

```diff
@@ -191,6 +204,13 @@ func resolveConfigurationFileFunc(policyGroup string) string {
 	}
 	return filePath
 }
+func GetConfigurationString(requestBytes []byte) string {
+	conf := string(requestBytes)
+	if strings.Contains(conf, "checksum") {
+		conf = strings.Split(conf, "checksum: ")[0]
```

Comment:
Can we rely on the fact that the checksum will always be the last part of the marshalled request?
The go yaml or json code might do that but there is no guarantee that it will happen for all requests that come in. We need to be really careful here.

Reply:
You are right. I modified it to parse the checksum wherever it is placed. If it is placed in the middle of the request, it will consider the first 64 characters after the `"Checksum: "` string